nix/egil-test-suite.nix

```
@@ -0,0 +1,122 @@
+{ version
+, homepage
+, downloadPage
+, changelog
+, maintainers
+, platforms
+}:
+{ lib
+, bash
+, docker
+, dockerTools
+, egil-scim-client
+, egil-test-server
+, filterSource
+, makeWrapper
+, openssl
+, stdenvNoCC
+, writeShellScript
+, ...
+}:
+
+let
+  inherit (lib)
+    makeSearchPath
+  ;
+  inherit (dockerTools)
+    pullImage
+  ;
```

Maybe all this is a little too verbose, I believe there are few more similar areas
```suggestion
  inherit (lib) makeSearchPath;
  inherit (dockerTools) pullImage;
```

---

Resolved with b009fac649ae036f939af95594108b3dc102be94